pkgs/development/tools/fdroidserver/default.nix

```
@@ -29,7 +30,7 @@
 }:
 
 let
-  version = "2.3a1";
+  version = "2.3a1-unstable-2024-09-04";
```

I think this should rather be

```suggestion
  version = "unstable-2024-09-04";
```

---

https://github.com/NixOS/nixpkgs/tree/master/pkgs#versioning